### ISSUE TITLE
Encumbranced Items in Loans - review

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2653]
Added encumbered record to loan: interface did say it had a restrict usage encumbrance in the fine print; this needs to be made more obvious; test allowed me to add encumbered item to loan regardless. Then in Review Loan Items, nothing shows up under Encumbrance field even though there is an encumbrance on the record.
@dustymc 
[Review Loan Items Encumbrance Error.pdf](https://github.com/ArctosDB/PG-migration-testing/files/4484483/Review.Loan.Items.Encumbrance.Error.pdf)

_Originally posted by @campmlc in https://github.com/ArctosDB/PG-migration-testing/issues/11#issuecomment-614369164_

**Comment by dustymc** [comment]
I'm not sure what this is asking for.

**Comment by campmlc** [comment]
We need to make it much more obvious through the interface and loan process that an item being added to loan has an encumbrance restricting usage, and if an encumbered item is added anyway, there should be  a warning/are you sure message and also some notification/column in the Review loan items that the part added is from an encumbered specimen.

**Comment by dustymc** [comment]
>more obvious through the interface and loan process 

I don't think this is compatible with bulk operations.

>also some notification/column in the Review loan items

That's always been there.

**Comment by dustymc** [comment]
>obvious through the interface
<img width="316" alt="Screen Shot 2020-05-07 at 10 04 20 AM" src="https://user-images.githubusercontent.com/5720791/81323401-2de23300-904a-11ea-82c1-21dd20c0cbd4.png">

**Comment by campmlc** [comment]
Yes, that seems to be fixed in test, although the interface has significant problems in usability compared to Oracle.
I still think that attempting to add an encumbered record to a loan should require a clicking a popup or some other form of acknowledement; for bulkloaded loans, perhaps a box that needs to be checked e.g. I wish to load these records even though I acknowledge some are encumbered.

**Comment by dustymc** [comment]
>significant problems in usability compared to Oracle

Do you mean the additional data added by user request or something else?

>require a clicking a popup

I can add that to any single interface, but I'm not sure I'll successfully remember to get them all nor to add it when the next cool idea comes along, and I certainly have no control over the UI in API-based applications. If this is something the community can agree on, finding a solution closer to the data would be useful.
